UNRELEASED.md

```diff
@@ -8,6 +8,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Enhancements
 
+- Tweaked colors in `TextField` and `Select` components ([#3378](https://github.com/Shopify/polaris-react/pull/3378))
```

Comment:
We've been excluding new design language changes from the changelog. You can remove this and just use the skip changelog label